lib/logstash/inputs/kafka.rb

```diff
@@ -216,6 +216,8 @@ class LogStash::Inputs::Kafka < LogStash::Inputs::Base
   config :sasl_oauthbearer_scope_claim_name, :validate => :string, :default => 'scope' # Kafka default
   # SASL login callback handler class
   config :sasl_login_callback_handler_class, :validate => :string
+  # Path to the jar containing client and all dependencies for SASL IAM authentication of specific cloud vendor
+  config :sasl_iam_jar_paths, :validate => :array
```

Comment:
The documentation for `sasl_iam_jar_paths` as a setting is missing in the asciidoc files. there's only the `AWS MSK IAM authentication` doc section explain how it is used for the AWS MSK use case.